profiles/common/themes/ec_resp_17/css/text_size_small.css

```diff
@@ -0,0 +1,4 @@
+/**
```

Comment:
@senseca  replied on JIRA task : 'Concerning "text_size_small.css", I didn't removed it in case it was used by some other sites.I don't know if it is necessary.'

OK you're right !!!